src/Plugin/TfaSetup/TfaTotpSetup.php

```diff
@@ -42,13 +43,13 @@ public function __construct(array $context) {
    */
   public function getSetupForm(array $form, FormStateInterface &$form_state) {
     $items = [
-      \Drupal::l('Google Authenticator (Android/iPhone/BlackBerry)', Url::fromUri('https://support.google.com/accounts/answer/1066447?hl=en'), array('attributes' => array('target'=>'_blank'))),
-      \Drupal::l('Authy (Android/iPhone)', Url::fromUri('https://www.authy.com/thefuture#install-now'), array('attributes' => array('target'=>'_blank'))),
-      \Drupal::l('Authenticator (Windows Phone)', Url::fromUri('http://www.windowsphone.com/en-us/store/app/authenticator/021dd79f-0598-e011-986b-78e7d1fa76f8'), array('attributes' => array('target'=>'_blank'))),
-      \Drupal::l('FreeOTP (Android)', Url::fromUri('https://play.google.com/store/apps/details?id=org.fedorahosted.freeotp'), array('attributes' => array('target'=>'_blank'))),
-      \Drupal::l('GAuth Authenticator (desktop)', Url::fromUri('https://github.com/gbraad/html5-google-authenticator'), array('attributes' => array('target'=>'_blank')))
+      Link::fromTextAndUrl('Google Authenticator (Android/iPhone/BlackBerry)', Url::fromUri('https://support.google.com/accounts/answer/1066447?hl=en', array('attributes' => array('target'=>'_blank')))),
```

Comment:
Please add an issue to pull these from the plugin systems - this will be a future work item (instead of hard coding them here)